### PR TITLE
bind 9.10.2-P2

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by DomT4** [comment]
Blurgh. Can you add a `version` line here please? Discovered this when I went to merge:

```
==> Applying patch
Applying: bind 9.10.2-P2
Applying: bind: update 2 bottle.
First, rewinding head to replay your work on top of it...
Applying: bind: update 2 bottle.
Downloaded to: /usr/local/var/homebrew/cache/bind-2.yosemite.bottle.tar.gz
```

File: Library/Formula/bind.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
-require "formula"
-
 class Bind < Formula
   desc "Implementation of the DNS protocols"
   homepage "https://www.isc.org/downloads/bind/"
-  url "http://ftp.isc.org/isc/bind9/9.10.2/bind-9.10.2.tar.gz"
-  sha1 "4ddb2670976c06af7e86352616383958d82c51ce"
+  url "http://ftp.isc.org/isc/bind9/9.10.2-P2/bind-9.10.2-P2.tar.gz"
```
Comment (DomT4) [inline Library/Formula/bind.rb]: This can be `https://` as well. @vszakats will be disappointed if I forget to point this out :wink:

**Comment by nijikon** [comment]
@DomT4 Everything fixed.

File: Library/Formula/bind.rb (hunk 1 above)
Comment (vszakats) [inline Library/Formula/bind.rb]: Indeed, as @DomT4 says :)

**Comment by DomT4** [comment]
@nijikon There's no `version` line in the formula. Am I missing something?

**Comment by nijikon** [comment]
This seem like a @BrewTestBot hiccup, can we restart?

**Comment by nijikon** [comment]
@mikemcquaid can you restart?

**Comment by mikemcquaid** [comment]
@nijikon FYI that happens if you force push after the job was initially triggered.

**Comment by mikemcquaid** [comment]
(restarted)

**Comment by nijikon** [comment]
@mikemcquaid any clue why this is failing on ML?

**Comment by DomT4** [comment]
Same thing happened the other day. I had to restart the build a couple times before the ML bot was able to fetch the file.

**Comment by aabdnn** [comment]
Any chance of this being fixed? This update to BIND fixes a critical vulnerability.

**Comment by mikemcquaid** [comment]
`curl` on 10.8 is getting `curl: (35) error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version`. @Homebrew/owners, any ideas?

File: Library/Formula/bind.rb [hunk 2]
```diff
@@ -1,10 +1,9 @@
-require "formula"
-
 class Bind < Formula
   desc "Implementation of the DNS protocols"
   homepage "https://www.isc.org/downloads/bind/"
-  url "http://ftp.isc.org/isc/bind9/9.10.2/bind-9.10.2.tar.gz"
-  sha1 "4ddb2670976c06af7e86352616383958d82c51ce"
+  url "https://ftp.isc.org/isc/bind9/9.10.2-P2/bind-9.10.2-P2.tar.gz"
```
Comment (mikemcquaid) [inline Library/Formula/bind.rb]: @nijikon Can you add `http://fossies.org/linux/misc/dns/bind9/9.10.2-P2/bind-9.10.2-P2.tar.gz` as a mirror? Current URL failing with a protocol error on 10.8.

**Comment by nijikon** [comment]
@mikemcquaid ```brew audit``` wants it to be https, what do we do now?

**Comment by DomT4** [comment]
Make it `https` :wink:

**Comment by nijikon** [comment]
OK, I just assumed that we want to make it http on purpose because of the curl issue.

**Comment by DomT4** [comment]
Na. The curl issue seems to be an issue with `ftp.isc.org` - doesn't always respond to requests.

**Comment by DomT4** [comment]
Can someone shed light on the critical vulnerability here? We're hitting OpenSSL issues elsewhere which we're looking at in #41615, so we're trying to hold off on merging things which depend on OpenSSL until after that.

**Comment by aabdnn** [comment]
The vulnerability in BIND is shown here:

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-4620

The issues with curl and openssl may be caused by the recent vulnerabilities in openssl, which have forced many sites to stop accepting older protocols and/or ciphers.

**Comment by DomT4** [comment]
Thanks for the CVE link. I'm tempted to merge this and then just add a revision tonight/tomorrow/etc when we decide how to handle the OpenSSL problems discussed in the other PR.

> which have forced many sites to stop accepting older protocols and/or ciphers.

The `curl` on recent OS X versions connects with `TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384` compared to `ECDHE-RSA-AES256-GCM-SHA384` for the most recent non-system curl built against an up-to-date OpenSSL.

**Comment by tdsmith** [comment]
+1 for merge

**Comment by DomT4** [comment]
Alright, merged in https://github.com/Homebrew/homebrew/commit/8239966b774317ac202b4ffd9e3cf064c370c790. Thanks @nijikon!

This may get a revision fairly quickly for the OpenSSL tweak, but we'll clear that hurdle when we get to it. For now at least, CVE handled.

**Pull request closed without merging**